go/errors/errors_test.go

```diff
@@ -97,63 +97,3 @@ func TestUnwrap(t *testing.T) {
 		})
 	}
 }
-
-func TestUnwrappedIs(t *testing.T) {
-	var err1 = errors.New("err1")
-
-	tcases := []struct {
-		err    error
-		target error
-		expect bool
-	}{
-		{
-			err:    nil,
-			expect: false,
-		},
-		{
-			err:    nil,
-			target: errors.New("err1"),
-			expect: false,
-		},
-		{
-			err:    errors.New("err1"),
-			target: errors.New("err1"),
-			expect: false,
-		},
-		{
-			err:    errors.New("err1"),
-			target: err1,
-			expect: false,
-		},
-		{
-			err:    err1,
-			target: err1,
-			expect: true,
-		},
-		{
-			err:    errors.Join(err1, errors.New("err2")),
-			target: err1,
-			expect: true,
-		},
-		{
-			err:    errors.Join(errors.New("err2"), err1),
-			target: err1,
-			expect: true,
-		},
-		{
-			err:    errors.Join(errors.New("err2"), errors.Join(errors.New("err3"), err1)),
-			target: err1,
-			expect: true,
-		},
-	}
-	for _, tcase := range tcases {
-		name := "nil"
-		if tcase.err != nil {
-			name = tcase.err.Error()
-		}
-		t.Run(name, func(t *testing.T) {
-			is := UnwrappedIs(tcase.err, tcase.target)
```

Comment:
@vmg if I replace `is := UnwrappedIs` with `is := errors.Is` then there actually is a test failure, but it's for testing whether `nil` error is `nil`, which returns true for `errors.Is` and false for `UnwrappedIs`. I'm perfectly happy with returning `true` so the change is good.

Reply:
Right, I did find that out. It seems like `errors.Is(nil, nil) == true` is very sensible behavior.